test/e2e/setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
In order to conduct E2E tests for OCO_PROMPT_MODULE=@commitlint, it was necessary to prepare an environment where the commitlint package is installed.

* Before running the E2E tests, we create node_modules according to the prepared package-lock.json (by setup.sh),
* We copy the node_modules with commitlint installed into the test environment within the E2E test suite,
* We are testing the commitlint command for each version of node_modules.